src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import {useState} from 'react';
 
 const App = () => {
+  const [chatEntry, setChatEntry] = useState(chatMessages)
```

Comment:
small nitpicky style note - I'd recommend naming the state variable `chatEntries` since it holds all of the chat messages. Calling it `chatEntry` makes it seem like the variable only holds one message which could be misleading.